metriken-exposition/src/parquet.rs

```diff
@@ -14,6 +14,9 @@ use parquet::format::{FileMetaData, KeyValue};
 
 use crate::snapshot::{HashedSnapshot, Snapshot};
 
+const DEFAULT_COMPRESSION: Compression = Compression::UNCOMPRESSED;
```

Comment:
If the `MsgpackToParquet` default has a compression level of 3 then so should the `ParquetOptions` default. It feels wrong to me that `MsgpackToParquet::new()` is different than `MsgpackToParquet::with_options(ParquetOptions::default())`

Reply:
This felt wrong to me too. I was hesitant to break the current behavior, but since this wound up being a breaking change to `MsgpackToParquet` anyway, I will address this before merge. Discussing with @mihirn offline to come to a consensus on this

Reply:
I'v made changes to address this.